src/components/Login/Login.module.css

```diff
@@ -0,0 +1,43 @@
+@import url('../../theme.module.css');
+
+.Login {
```

Comment:
For consistency throughout this file 
```suggestion
.login {
```
You will need to update the name everywhere it's used

Reply:
yes, makes more sense